decidim-admin/config/routes.rb

```diff
@@ -39,7 +39,11 @@
       end
     end
 
-    resources :officializations, only: [:new, :create, :index, :destroy], param: :user_id
+    resources :officializations, only: [:new, :create, :index, :destroy], param: :user_id do
+      member do
+        get :show_email
+      end
```

Comment:
As it has only one action, it could be rewritten to one line: `get :show_email, on: :member`